core/templates/pages/admin-page/services/admin-misc-tab-backend-api.service.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to revert/change admin-misc-tab properties
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { AdminPageConstants } from 'pages/admin-page/admin-page.constants';
+
+ @Injectable({
+   providedIn: 'root'
+ })
+export class AdminMiscTabBackendApiService {
+  constructor(private http: HttpClient) {}
+
+  private _flushCache ():Promise<void> {
```

Comment:
Why these methods are private? (I see a public method with the same name below, I would suggest having just one public function.)

Reply:
done!